CET - critical events, notifications, and Jira integration (#1351)
harness/developer-hub: docs/service-reliability-management/continuous-error-tracking/event-explorer/hidden-events.md

---
title: Hidden events
description: Mark events as hidden.
sidebar_position: 3
---

Continuous Error Tracking allows you to hide insignificant or known issues, as well as resolved events, from the Events page in order to reduce noise.

If you identify an issue as minor or known and mark it as "hidden", it will be moved to the **Hidden Events** page. A hidden event no longer appears on the Events page. You can restore a hidden event any time.

## Hide events

To hide events:

1. In your Harness project, navigate to **Service Reliability Management** and select **Code Errors**.    
   A list of all the services monitored by Continuous Error Tracking is displayed.

2. Select a monitored service for which you want to view the events.

3. On the Events page, select the events that you want to mark as hidden and then select **Hide Events**.

>    The selected events will be moved to the **Hidden Events** page and no longer appear on the Events page.

Reviewer: ```suggestion
   The selected events are moved to the **Hidden Events** page and no longer appear on the Events page.
```

   ![Hide an event](./static/hide-events.gif)

## Restore hidden events

To restore hidden events:

1. On the **Hidden Events** page, select the events that you want to restore and then select **Restore**.

>    The selected events will be moved to the **Events** page and will no longer appear on the **Hidden Events** page.

Reviewer: ```suggestion
   The selected events are moved to the **Events** page and no longer appear on the **Hidden Events** page.
```
